### PR TITLE
fix(core-select): do not send empty fields / wrong operation parameter name

### PR DESCRIPTION
**Pull request by GuillaumeNury**
## Description

* do not send empty fields
* wrong operation parameter name

-----

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2816/storybook

File: packages/ng/core-select/user/users.directive.ts [hunk 1]
```diff
@@ -130,7 +130,7 @@ export class LuCoreSelectUsersDirective<T extends LuCoreSelectUser = LuCoreSelec
 		const params = {
 			fields: this.#userFields,
 			...(this._filters() ?? {}),
-			...(this._operationIds() ? { operationIds: this._operationIds().join(',') } : {}),
+			...(this._operationIds() ? { operations: this._operationIds().join(',') } : {}),
```
Comment (shajz) [inline packages/ng/core-select/user/users.directive.ts:133]: wasn't this fixed somewhere already? I'm having "déjà vu" :D

**Review by shajz — APPROVED** [review]

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/17344/ui-diff)

File: packages/ng/core-select/user/users.directive.ts (hunk 1 above)
Comment (GuillaumeNury) [reply packages/ng/core-select/user/users.directive.ts:133]: it was fixed a few lines above, for regular options. The present fix is for the "Me: " option 🙈

**Pull request merged**